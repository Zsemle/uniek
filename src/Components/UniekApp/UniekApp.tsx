import React from 'react';
import './UniekApp.css';
import PlanPage from '../PlanPage/PlanPage';

const UniekApp:React.FC = ():JSX.Element => (
  <div className="uniek-app">
    <PlanPage />
  </div>
);

export default UniekApp;
