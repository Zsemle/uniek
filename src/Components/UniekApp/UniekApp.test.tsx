import React from 'react';
import { render, screen } from '@testing-library/react';
import UniekApp from './UniekApp';

test('renders learn react link', () => {
  render(<UniekApp />);
  const linkElement = screen.getByText(/learn react/i);
  expect(linkElement).toBeInTheDocument();
});
