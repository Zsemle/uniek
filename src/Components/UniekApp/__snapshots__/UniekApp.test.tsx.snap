// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the app layout component renders the component 1`] = `
<Fragment>
  <CssBaseline />
  <div
    className="uniek-app"
  >
    <AppHeader />
    <PlanPage />
  </div>
</Fragment>
`;
